src/hooks/useAuth.utils.ts

```diff
@@ -147,7 +150,7 @@ export async function createConnection(
   chainId: ChainId
 ) {
   try {
-    const data = await connection.connect(providerType, chainId)
+    const data = await connection.connect(providerType as any, chainId)
```

Comment:
why is this as any needed?

Reply:
There was a development issue that was later not needed, I've removed it.